ToDoAppWithCoreData/Controllers/TableViewController.swift

```diff
@@ -0,0 +1,154 @@
+
+import UIKit
+import CoreData
+
+
+class TableViewController: UITableViewController, ListViewControllerDelegate {
+    
+    var myList: Array<AnyObject> = []
```

Comment:
こういうのもなるべくprivateにしておいた方が、無理やりアクセスしなくなるのでおすすめです
```suggestion
    private var myList: Array<AnyObject> = []
```

Reply:
ところで、AnyObjectじゃ無くてNSManagedObjectにしておけばいろんなところでas!しなくて済むのでは？